maat-court-data-api/src/main/java/gov/uk/courtdata/reporder/controller/RepOrderController.java

```diff
@@ -116,7 +116,6 @@ public ResponseEntity<Object> findByRepIdAndVehicleOwner(@PathVariable int repId
     @StandardApiResponse
     public ResponseEntity<RepOrderDTO> create(@Valid @RequestBody CreateRepOrder createRepOrder) {
         log.debug("Create Rep order request");
-        maatIdValidator.validateNotExists(createRepOrder.getRepId());
```

Comment:
Should we remove the validateNotExists method from MaatIdValidator class as well?

Reply:
Thanks @venkat980 . It might use in future, hence not removed